<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>内存清理工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxMemory">
      <property name="title">
       <string>内存状态</string>
      </property>
      <layout class="QGridLayout" name="gridLayoutMemory">
       <item row="0" column="1">
        <widget class="QLabel" name="lbPhysicalValue">
         <property name="text">
          <string>0.00 GB / 0.00 GB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelPhysical">
         <property name="text">
          <string>物理内存:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelSystemWS">
         <property name="text">
          <string>系统工作集:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lbVirtualValue">
         <property name="text">
          <string>0.00 GB / 0.00 GB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lbSystemWSValue">
         <property name="text">
          <string>0.00 GB (共享 0.00 GB) / 0.00 GB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelVirtual">
         <property name="text">
          <string>虚拟内存:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="pbPhysical">
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QProgressBar" name="pbVirtual">
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QProgressBar" name="pbSystemWS">
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxOptions">
      <property name="title">
       <string>清理选项</string>
      </property>
      <layout class="QGridLayout" name="gridLayoutOptions">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbWorkingSet">
         <property name="text">
          <string>Working Set</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbSystemWorkingSet">
         <property name="text">
          <string>System Working Set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbStandbyList">
         <property name="text">
          <string>Standby List</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbStandbyListLow">
         <property name="text">
          <string>Standby List (Low Priority)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbModifiedPageList">
         <property name="text">
          <string>Modified Page List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="cbCombinePageList">
         <property name="text">
          <string>Combine Page List</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxControl">
      <property name="title">
       <string>控制</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutControl">
       <item>
        <widget class="QPushButton" name="btnCleanNow">
         <property name="text">
          <string>立即清理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbAutoClean">
         <property name="text">
          <string>自动清理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelInterval">
         <property name="text">
          <string>间隔(秒):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbInterval">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>3600</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lbLastUpdate">
      <property name="text">
       <string>最后更新: -</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
